Description: Update shebang to use Python 3
Author: Josue Ortega <josue@debian.org>
Last-Update: 2019-09-08
Forwarded: not-needed

--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 import os
 import sys
 
--- a/bin/mido-connect
+++ b/bin/mido-connect
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Forward all messages from one or more ports to server.
 """
--- a/bin/mido-ports
+++ b/bin/mido-ports
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 List available PortMidi ports.
 """
--- a/bin/mido-serve
+++ b/bin/mido-serve
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Serve one or more output ports. Every message received on any of the
 connected sockets will be sent to every output port.
--- a/examples/sockets/forward_ports.py
+++ b/examples/sockets/forward_ports.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Forward all messages from one or more ports to server.
 
--- a/examples/sockets/serve_ports.py
+++ b/examples/sockets/serve_ports.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Serve one or more output ports.
 
--- a/examples/sockets/simple_client.py
+++ b/examples/sockets/simple_client.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Simple client that sends program_change messages to server at timed
 intervals.
--- a/examples/sockets/simple_server.py
+++ b/examples/sockets/simple_server.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Simple server that just prints every message it receives.
 
--- a/examples/midifiles/create_midi_file.py
+++ b/examples/midifiles/create_midi_file.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Create a new MIDI file with some random notes.
 
--- a/examples/midifiles/play_midi_file.py
+++ b/examples/midifiles/play_midi_file.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Play MIDI file on output port.
 
--- a/examples/midifiles/print_midi_file.py
+++ b/examples/midifiles/print_midi_file.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Open a MIDI file and print every message in every track.
 
--- a/examples/ports/input_filter.py
+++ b/examples/ports/input_filter.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Receive messages on the input port and print messages with a specific
 channel (defaults to 0).
--- a/examples/ports/list_ports.py
+++ b/examples/ports/list_ports.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 List available ports.
 
--- a/examples/ports/multi_receive.py
+++ b/examples/ports/multi_receive.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Receive messages from multiple ports.
 """
--- a/examples/ports/nonblocking_receive.py
+++ b/examples/ports/nonblocking_receive.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Example of non-blocking reception from input port.
 """
--- a/examples/ports/receive.py
+++ b/examples/ports/receive.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Receive messages from the input port and print them out.
 """
--- a/examples/ports/send.py
+++ b/examples/ports/send.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """
 Send random notes to the output port.
 """
